rust/noosphere/tests/gateway.rs

```diff
@@ -12,23 +16,21 @@ use tokio::io::AsyncReadExt;
 use tokio_stream::StreamExt;
 use url::Url;
 
-use noosphere_api::route::Route;
+use noosphere_core::api::v0alpha1;
```

Comment:
Expecting `v0alpha2` here, an internal detail to know which route has/not changed between versions